packages/appstore/src/assets/svgs/wallets/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReadyToUpdateWalletsIcon from './ic-wallet-upgrade-form.svg';
+
+export const WalletsImagesList = {
+    ReadyToUpdateWalletsIcon,
+};
+
+type WalletsImageProps<T> = {
+    banner: T;
+    className?: string;
+};
+
+const WalletsImage = ({ banner, className }: WalletsImageProps<keyof typeof WalletsImagesList>) => {
+    const Component = WalletsImagesList[banner] as React.ElementType;
+
+    return <Component className={className} data-testid={banner} />;
```

Comment:
use the same approach of this PR: https://github.com/binary-com/deriv-app/blob/8d8f7bfa06ba80cd75e2572562576125174b0448/packages/appstore/src/assets/svgs/wallets/index.tsx

Reply:
Done 😃 👍🏼